code/game/turfs/open/floor/plating/asteroid.dm

```diff
@@ -115,10 +124,6 @@
 	. = ..()
 	set_basalt_light(src)
 
-/turf/open/floor/plating/asteroid/getDug()
```

Comment:
Someone omitted to type the `/basalt` node before `/getDug()`, normal asteroid turfs already have their definition of the proc and aren't lit iirc.